packages/cli/src/commands/proxy.ts

```diff
@@ -0,0 +1,41 @@
+import * as signale from 'signale';
+import { CommandModule } from 'yargs';
+import { createMultiProcessPrism, CreatePrismOptions, createSingleProcessPrism } from '../util/createServer';
+import getHttpOperations from '../util/getHttpOperations';
+import options from './options';
+
+const proxyCommand: CommandModule = {
+  describe: 'Start a proxy server with the given spec file',
+  command: 'proxy <upstream> <spec>',
+  builder: yargs =>
+    yargs
+      .positional('upstream', {
+        description: 'An address for the destination server.',
+        type: 'string',
+      })
+      .positional('spec', {
+        description: 'Path to a spec file. Can be both a file or a fetchable resource on the web.',
+        type: 'string',
+      })
+      .middleware(async argv => (argv.operations = await getHttpOperations(argv.spec!)))
+      .fail((msg, err) => {
+        if (msg) yargs.showHelp();
+        else signale.fatal(err.message);
+
+        process.exit(1);
+      })
+      .options(options),
```

Comment:
Sharing all options between mock and proxy is a little confusing as we do not want or need --dynamic. Can we have some shared options and some specific to the command?

Reply:
Yes we can